frontend/components/stories/feature/query-filters/multiselect-filter.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { queryBuilderRemote } from '@/components/model/query-builder-core/query-builder-remote';
+import { MultiSelectFilter } from '@/components/feature/query-filters/multiselect-filter';
+import { ConfigProvider, PortalConfig } from '@/components/model/applications-config';
+
+const config: PortalConfig = {
+  variant_entity: {
+    app_id: 'variant_entity',
+    aggregations: [
+      { key: 'chromosome', type: 'multiple' },
+      { key: 'filter', type: 'multiple' },
+      { key: 'zygosity', type: 'multiple' },
+      { key: 'impact_score', type: 'multiple' },
+      { key: 'variant_class', type: 'multiple' },
+      { key: 'symbol', type: 'multiple' },
+    ],
+  },
+};
+
+const meta = {
+  title: 'Feature/Query Filters Multi Select',
```

Comment:
@francisl Tu peux rajouter un `/` ca va mettre le multi select et toggle sous un folder

```suggestion
  title: 'Feature/Query Filters/Multi Select',
```